lib/fastlane/plugin/notarize/actions/notarize_action.rb

```diff
@@ -7,6 +7,7 @@ def self.run(params)
         package_path = params[:package]
         bundle_id = params[:bundle_id]
         try_early_stapling = params[:try_early_stapling]
+        disable_detailed_log = params[:disable_detailed_log]
```

Comment:
Should we go with a simpler name, like `print_log`?

Reply:
In the `print_log` case, the default value would be `true` though.

Reply:
yes lets go with print_log, i will change it

Reply:
Okay changed it!